src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1,31 +1,14 @@
-INSERT INTO tab_rate_generic_parameters(calculation_method,formula_rate,path_id,parameter_name,alias) VALUES('OntogenyFactors','TableFormulaWithXArgument_OntogenyFactorGI',240,'Ontogeny factor GI table','Table');
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5638 AND container_type='ORGAN' AND container_name='Breasts' AND parameter_name='Specific blood flow rate';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5638 AND container_type='ORGAN' AND container_name='Breasts' AND parameter_name='Volume';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5639 AND container_type='ORGAN' AND container_name='Myometrium' AND parameter_name='Specific blood flow rate';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5639 AND container_type='ORGAN' AND container_name='Myometrium' AND parameter_name='Volume';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5640 AND container_type='ORGAN' AND container_name='Endometrium' AND parameter_name='Specific blood flow rate';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5640 AND container_type='ORGAN' AND container_name='Endometrium' AND parameter_name='Volume';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5641 AND container_type='ORGAN' AND container_name='PlacentaMaternal' AND parameter_name='Specific blood flow rate';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5641 AND container_type='ORGAN' AND container_name='PlacentaMaternal' AND parameter_name='Volume';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5642 AND container_type='ORGAN' AND container_name='PlacentaFetal' AND parameter_name='Specific blood flow rate';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5642 AND container_type='ORGAN' AND container_name='PlacentaFetal' AND parameter_name='Volume';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5643 AND container_type='ORGAN' AND container_name='Fetus' AND parameter_name='Volume';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5644 AND container_type='ORGAN' AND container_name='AmnioticFluid' AND parameter_name='Volume';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5645 AND container_type='ORGAN' AND container_name='ArterialBlood_UmbilicalCord' AND parameter_name='Volume';
+UPDATE tab_container_parameters SET is_changed_by_create_individual=1 WHERE container_id=5646 AND container_type='ORGAN' AND container_name='VenousBlood_UmbilicalCord' AND parameter_name='Volume';

```

Comment:
This is in fact the only change - setting is_changed_by_create_individual=1 for some pregnancy-specific volumes and blood flows